### ISSUE TITLE
0 failed files found

### ISSUE THREAD

**Issue by badbake** [TickTockBent/spudchecker#1]
![image](https://github.com/user-attachments/assets/dc622b97-61d4-42e3-a805-a4f46d6970e8)

**Comment by TickTockBent** [comment]
Agreed, it shouldn't make a file when no failed validations are found. I'll fix that in the next release
